Quickstart/01-Login/MvcApplication/MvcApplication/Startup.cs

```diff
@@ -57,7 +57,8 @@ public void Configuration(IAppBuilder app)
                 {
                     NameClaimType = "name"
                 },
-
+                
+                // SameSiteCookieManager is not a built in constructor but is a separate class created as part of the demo
```

Comment:
I am not sure how this comment helps, the user could just click on the method and go to the definition to see it is our own code if they are inside the project looking at the code?

This class is also part of official .NET documentation: https://docs.microsoft.com/en-us/aspnet/samesite/owin-samesite

I think it might make more sense to add a comment on the [quickstart](https://github.com/auth0/docs/blob/master/articles/quickstart/webapp/aspnet-owin/_includes/_login.md) rather then the sample in here. Might also be worth refering to the article from microsoft mentioned above,

Reply:
Hi Frederik,

Yes, that makes more sense :) I just put it there because the customer pointed that out. I think that it will be helpful if we mention it somewhere. Thank you!